From 4ebf338a054bb846f44f5ba98f8ff46ce34e9429 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Thu, 29 Apr 2021 08:46:28 -0500
Subject: [PATCH 83/83] dt-bindings: imx8mn: Fix reference to IMX8MN_CLK_I2C4

The reference to IMX8MN_CLK_I2C4 should be unique. Unfortunately,
it's duplicating a reference to another clock.  Fix this by making
it unique.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 include/dt-bindings/clock/imx8mn-clock.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/dt-bindings/clock/imx8mn-clock.h b/include/dt-bindings/clock/imx8mn-clock.h
index f4a18541f987..afd99a31124e 100644
--- a/include/dt-bindings/clock/imx8mn-clock.h
+++ b/include/dt-bindings/clock/imx8mn-clock.h
@@ -124,7 +124,7 @@
 #define IMX8MN_CLK_I2C1				105
 #define IMX8MN_CLK_I2C2				106
 #define IMX8MN_CLK_I2C3				107
-#define IMX8MN_CLK_I2C4				118
+#define IMX8MN_CLK_I2C4				108
 #define IMX8MN_CLK_UART1			109
 #define IMX8MN_CLK_UART2			110
 #define IMX8MN_CLK_UART3			111
-- 
2.17.1

